bg_remove_iterative.py

```
@@ -0,0 +1,430 @@
+  3. Snapshot → agent inspects result
+  4. Repeat until no BG pixels remain or max iterations hit
+
+Usage:
+    python bg_remove_iterative.py --input path/to/image.png --output-dir path/to/output/
+"""
+import argparse, socket, json, base64, sys
+
+# ── transport ───────────────────────────────────────────────────────────────
+
+def cmd(t, params=None):
+    s = socket.socket(); s.settimeout(30)
```

Ruff (0.15.9)

[error] Multiple statements on one line (semicolon)

(E702)